### DISCUSSION TITLE
Extend Presets by Ability to stage part of conversation.

### DISCUSSION THREAD

**Discussion by juicesharp** [danny-avila/LibreChat#1589] [Feature Requests & Suggestions]
In prompt engineering area quite a lot of techniques like COT, few shots e.t.c. require to run a conversation from a specific point where some messages User/Assistant are already added into the chat in advance (staged).
![2024-01-18_20-10-32](https://github.com/danny-avila/LibreChat/assets/7575510/47efd711-3692-4dbe-9647-18ea40771f25)  It would be nice to extend the current presets with a such ability that would allow to use librechat in place of Open AI Playground as Prompt IDE.

**Comment by danny-avila** [comment]
Thanks I agree! Adding/removing messages at any point as well as conversation templates are planned!
